src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -854,6 +860,49 @@ public function collectFromEntityProviders($filePath, Module $module = null)
         return $entities;
     }
 
+    /**
+     * Extracts translations for ORM fields
+     *
+     * @param string $filePath
+     * @return array
+     */
+    public function collectFromORM($filePath)
+    {
+        $entities = [];
+        $classes = ClassInfo::classes_for_file($filePath);
+        foreach ($classes as $class) {
+            // Skip non-implementing classes
+            if (!class_exists($class ?? '')) {
```

Comment:
Is the null coalesce needed here?

Reply:
I don't know. I have copied the base of the method from https://github.com/silverstripe/silverstripe-framework/blob/4.12/src/i18n/TextCollection/i18nTextCollector.php#L818.